### PR TITLE
Code only tag

### LINKED ISSUE #109: Allow for creating code that be placed only in code_only file

### PR DESCRIPTION
**Pull request by plutasnyy**

### CODE REVIEW

File: sst/src/exporter/factory.py [hunk 1]
```diff
@@ -18,9 +19,9 @@ def markdown_exporter_with_preprocessors(execute_enabled: bool) -> Exporter:
 
     config = Config()
     for apply_configuration in [
-        configure_tag_removal_preprocessor,
+        configure_tag_remove_preprocessor_to_hide_output,
+        configure_tag_remove_preprocessor_to_remove_cell,
         configure_copyright_regex_removal_preprocessor,
-        configure_extract_outputs_preprocessor
```
Comment (Adamage) [inline sst/src/exporter/factory.py:23]: its not needed?

**Review by Adamage — COMMENTED** [review]
Looks good to meee

File: sst/src/exporter/factory.py (hunk 1 above)
Comment (plutasnyy) [reply sst/src/exporter/factory.py:23]: I removed all configurations that set `.enabled=True` flag, while we initialize the preprocessor using:
`exporter.register_preprocessor(preprocessor, enabled=True)` it is not necessary

**Review by Adamage — APPROVED** [review]

File: sst/README.md [hunk 2]
```diff
@@ -74,6 +74,18 @@ for _ in range(1000):
 # sst_hide_output
 ```
 
+#### Hiding cells
```
Comment (dirkbrink) [inline sst/README.md:77]: I noticed that the markdown cell (comment delimited) cell above the code cell with the tag will also be hidden from the code only output...worth mentioning that in the README.

**Review by dirkbrink — CHANGES_REQUESTED** [review]

File: sst/README.md (hunk 2 above)
Comment (dirkbrink) [reply sst/README.md:77]: Oh never mind I'm being dumb...it just applies to both comment cells and code cells - the difference being you don't need the "#" in the comment cell

**Review by dirkbrink — APPROVED** [review]

**Pull request merged**